### DISCUSSION TITLE
IPv6 netflow transport

### DISCUSSION THREAD

**Discussion by rpravm** [akvorado/akvorado#1011] [Q&A]
Trying to send flows from Juniper MX to Akvorado instance over IPv6. 
I can see traffic on my NIC, but Akvorado do not show any received flows, nor write errors to inlet logs.

Is it supported?

**Comment by vincentbernat** [comment]
Yes, it should be. However, by default Docker does not work over IPv6. See https://github.com/akvorado/akvorado/discussions/922.
